core/arch/arm/plat-rockchip/common.h

```diff
@@ -46,4 +46,7 @@
 #define BITS_WITH_WMASK(bits, msk, shift) \
 				(SHIFT_U32(bits, shift) | BITS_WMSK(msk, shift))
 
+#define __sramfunc __attribute__((section(".sram.text"))) \
```

Comment:
This is not 100% proof. Static data allocated by the function may still end up in `.rodata`, `.data` or `.bss`.

The technique used to separate unpaged and init code/rodata from paged code/rodata can be used here too.